src/services/ui/src/api/useGetItem.ts

```diff
@@ -44,15 +46,21 @@ export const useGetItem = (
 
 export const useGetItemCache = (id: string) => {
   const queryClient = useQueryClient();
+  const user = useGetUser();
   const data = (() => {
     const data = queryClient.getQueryCache().find(["record", id])?.state
       .data as opensearch.main.ItemResult;
     return data?._source;
   })();
+  // const actions = getAvailableActions(user.data?.user, data)
+  const actions = (() => {
+    if (!user.data?.user) return [];
+    return getAvailableActions(user.data?.user, data);
```

Comment:
I didn't see any reason that we were bundling this in with the item itself, so we should be able to use the `getAvailableActions` utility func from wherever we need the actions.